### ISSUE TITLE
Package.getStylesheetsPath is deprecated.

### ISSUE THREAD

**Issue by karabenemsi** [camel-chased/clean2#9]
Store package style sheets in the `styles/` directory instead of `stylesheets/` in the `Clean2` package

```
Package.getStylesheetsPath (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:460:9)
Package.getStylesheetPaths (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:471:32)
Package.loadStylesheets (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:453:38)
<unknown> (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:183:19)
Package.measure (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:163:15)
Package.load (C:\Users\Florian\AppData\Local\atom\app-0.200.0\resources\app.asar\src\package.js:177:12)
```

**Comment by duub** [comment]
It has been fixed in the 0.14.0 release

**Comment by karabenemsi** [comment]
Jep, atom fucked up updating. Sorry
